metrics/errors.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  name: 'enables JavaScript Sentry error metrics',
+  startDate: '2015-01-01',
+  subjectAttributes: ['uuid', 'sampleRate'],
+  independentVariables: ['sentryEnabled'],
+  eligibilityFunction: function (subject) {
+    return !! subject.sampleRate && subject.uuid && this.bernoulliTrial(subject.sampleRate, subject.uuid)
```

Comment:
So we are going to pass in the sample rate, I'm assuming it comes from the `/config` endpoint? Would it make sense to skip `/config` and just define the sample rate here and update as needed?

Not sure what is easiest.

So happy to see `uuid`.